scripts/install.sh

```diff
@@ -143,11 +150,23 @@ install_tool() {
     esac
 }
 
+install_dotnet_x64_for_arm() {
+    log INFO "downloading and installing dotnet x64 binaries in arm machines"
```

Comment:
This is temporary method until we fix the build pipelines to produce arm64 binaries for bridge. Thank you.

Reply:
Works in M1 Mac with this temporary fix for arm machines.

curl -fsSL https://raw.githubusercontent.com/hsubramanianaks/Bridge-To-Kubernetes/fix-script-dotnet-6/scripts/install.sh | bash

<img width="583" alt="Screen Shot 2023-02-23 at 2 17 15 AM" src="https://user-images.githubusercontent.com/10390606/220755163-b4f6fe7c-bb14-438b-8eeb-d0f4500c01a9.png">


<img width="1347" alt="Screen Shot 2023-02-23 at 2 16 09 AM" src="https://user-images.githubusercontent.com/10390606/220755057-3ab7443b-1268-484c-b462-1b914f5cdef8.png">

<img width="784" alt="Screen Shot 2023-02-23 at 2 18 07 AM" src="https://user-images.githubusercontent.com/10390606/220755323-4724c557-784b-49e4-b998-80bdd080cd02.png">